.buildkite/pipeline.yml

```diff
@@ -167,18 +167,18 @@ steps:
         artifact_paths: "sphere_held_suarez_rhotheta/*"
 
       - label: ":computer: held suarez (ρe) equilmoist"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_equilmoist --turbconv const_diffusivity --moist equil --forcing held_suarez --microphy 0M --job_id sphere_held_suarez_rhoe_equilmoist --regression_test true --dt 200 --t_end 432000"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_equilmoist --turbconv const_diff --moist equil --forcing held_suarez --microphy 0M --job_id sphere_held_suarez_rhoe_equilmoist --regression_test true --dt 200 --t_end 432000"
         artifact_paths: "sphere_held_suarez_rhoe_equilmoist/*"
 
       - label: ":computer: held suarez (ρe_int) equilmoist"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_int_equilmoist --turbconv const_diffusivity --energy_name rhoe_int --forcing held_suarez --moist equil --microphy 0M --job_id sphere_held_suarez_rhoe_int_equilmoist --regression_test true --dt 200 --t_end 432000"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_int_equilmoist --turbconv const_diff --energy_name rhoe_int --forcing held_suarez --moist equil --microphy 0M --job_id sphere_held_suarez_rhoe_int_equilmoist --regression_test true --dt 200 --t_end 432000"
```

Comment:
I may be careless in the previous PR review, but I think we want vertical diffusion a separate option as turbconv, meaning that we could have edmf + 3d-const-visc + vertical-diff all turned on

Reply:
I'm certainly open to this (I think this was what I did to start with), but I think I shared the same flag for constant diff with edmf based off @yairchn's suggestion. If we split it, then what should we call the different options?

Reply:
not sure I see the logic of having all `edmf + 3d-const-visc + vertical-diff` but sure why not

Reply:
Yea, this is something Tapio mentioned several times. I would just call it something like `vert_diff` for vertical only diffusion and `turbulence` as a place holder if we want to add second order diffusion, and keep `turbconv` for edmf